README.md

```diff
@@ -1,122 +1,101 @@
+# .NET Interactive Documentation 
 
-# .NET Interactive <img src ="https://user-images.githubusercontent.com/2546640/56708992-deee8780-66ec-11e9-9991-eb85abb1d10a.png" width="80px" alt="dotnet bot in space" align ="right">
+## What is .NET Interactive?
 
-[![Discord](https://img.shields.io/discord/732297728826277939?label=discord)](https://discord.gg/3pvut9YujN) [![Build Status](https://dev.azure.com/dnceng-public/public/_apis/build/status/dotnet/interactive/interactive-ci?branchName=main)](https://dev.azure.com/dnceng-public/public/_build?definitionId=71&branchName=main) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dotnet/interactive/main?urlpath=lab) 
+.NET Interactive is an engine that can run multiple languages and share variables between them. Languages currently supported include: 
 
-.NET Interactive takes the power of .NET and embeds it into various interactive experiences. Share code, explore data, write, and learn across your apps in ways you couldn't before.
-
-* [Notebooks](#notebooks-with-net): Jupyter, nteract, and Visual Studio Code 
-* [Code bots](https://github.com/CodeConversations/CodeConversations)
-* Devices like [Raspberry Pi](https://www.raspberrypi.org/)
-* Embeddable script engines
-* [REPLs](https://github.com/jonsequitur/dotnet-repl)
+- C# 
+- F#
+- PowerShell
+- JavaScript
+- SQL 
+- KQL (Kusto Query Language)
+- HTML*
```

Comment:
Value sharing is actually supported in the browser kernel and might show up in the notebook as well. Given that, I wonder if this detail is important or entirely accurate.

Reply:
I'd like the docs to reflect the current functionality - the browser kernel isn't currently available in stable correct? Are you ok with tabling this detail until it is available and then we remove the * from HTML? (unless I'm wrong about the browser kernel available in stable today).

Reply:
It's fine either way and I'm probably just being pedantic. 

The browser kernel is in preview as are all of the NuGet packages, including SQL and KQL. It's definitely less mature than others, but it can be loaded into the notebook and used right now. It's not the default HTML kernel that's in the box, though.